includes/admin/reporting/reports-callbacks.php

```diff
@@ -118,7 +118,7 @@ function edd_overview_sales_earnings_chart() {
 
 		// Loop through each date there were sales/earnings, which we queried from the database.
 		foreach ( $earnings_results as $earnings_result ) {
-			$date_of_db_value = new DateTime( $earnings_result->date, $timezone );
+			$date_of_db_value = edd_get_edd_timezone_equivalent_date_from_utc( EDD()->utils->date( $earnings_result->date ) );
```

Comment:
I think this change and line 146 are really from a different issue/PR and should be removed from this? Otherwise the PR looks good.

Reply:
@zkawesome I think robin is right here, looks like this is from the #9278 PR